ui/const/icons.ts

```diff
@@ -0,0 +1,13 @@
+export const COIN_ICONS = {
+  ADA: '/coins/ADA.svg',
```

Comment:
I don't forsee us using ADA or BNB or BTW (given we have WBTC) since those are native to other chains